pkgs/misc/urbit/sources.nix

```diff
@@ -0,0 +1,174 @@
+# This file has been generated by Niv.
```

Comment:
What the hell???

Reply:
I checked full repo before commit and there are other generated sources. Anyways I keep this from upstream to ease maintenance, I could get rid of it.

Reply:
The artifacts of a code generation should be generated instead of injected.
E.g. it is preferable to include a .yacc file (and bison as build input) instead of including a post-processed C file.

Indeed it is very hard to review that code.